### DISCUSSION TITLE
How to use normal mode with vimscript

### DISCUSSION THREAD

**Discussion by praxder** [JetBrains/ideavim#510] [Q&A] [answered]
I'm loving the Ideavim features and am so excited where it's going. I see on the Vimscript [roadmap](https://github.com/JetBrains/ideavim/blob/master/vimscript-info/VIMSCRIPT_ROADMAP.md) that `normal` mode command was added in release 1.9.0. The issue is that I can't figure out how to use it. Anytime I try to use a `normal` mode command I get the error "VIM - E183: User defined commands must start with an uppercase letter", which makes me think that `normal` mode isn't recognized. I'm using the latest version of Android Studio and the IdeaVim plugin (1.9.2).

Can someone help clarify or give me an example of how to use `normal` mode in Vimscript?

**Comment by AlexPl292** [comment, marked as answer]
Hi, we had to disable the normal command because of some issues in IdeaVim and platform. You can enable it for you by adding `set noideadelaymacro` to your `~/.ideavimrc`. The issues were about undone things after `:normal` and macros.
